airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -14,210 +14,258 @@
 #
 
 """
-This module contains a sqoop 1 hook
+This module contains a sqoop 1.x hook
 """
 
-from airflow.hooks.base_hook import BaseHook
-from airflow.exceptions import AirflowException
-
 import logging
 import subprocess
 
+from airflow.exceptions import AirflowException
+from airflow.hooks.base_hook import BaseHook
+
 log = logging.getLogger(__name__)
 
 
 class SqoopHook(BaseHook):
     """
     This Hook is a wrapper around the sqoop 1 binary. To be able to use te hook
     it is required that "sqoop" is in the PATH.
-    :param hive_home: (from json) The location of hive-site.xml
-    :type hive_home: str
-    :param job_tracker: (from json) <local|jobtracker:port> specify a job tracker
+    :param job_tracker: (from json) specify a job tracker local|jobtracker:port
     :type job_tracker: str
     :param namenode: (from json) specify a namenode
     :type namenode: str
-    :param lib_jars: (from json) specify comma separated jar files to
-        include in the classpath.
+    :param lib_jars: (from json) specify comma separated jar
+        files to include in the classpath.
     :type lib_jars: str
-    :param files: (from json) specify comma separated files to be copied
-        to the map reduce cluster
+    :param files: (from json) specify comma separated files to be copied to
+        the map reduce cluster
     :type files: (from json) str
     :param archives: (from json)  specify comma separated archives to be
         unarchived on the compute machines.
     :type archives: str
     """
-    def __init__(self, conn_id='sqoop_default'):
-        conn = self.get_connection(conn_id)
-        self.hive_home = conn.extra_dejson.get('hive_home', None)
-        self.job_tracker = conn.extra_dejson.get('job_tracker', None)
-        self.namenode = conn.extra_dejson.get('namenode', None)
-        self.lib_jars = conn.extra_dejson.get('libjars', None)
-        self.files = conn.extra_dejson.get('files', None)
-        self.archives = conn.extra_dejson.get('archives', None)
-        self.conn = conn
+
+    def __init__(self, conn_id='sqoop_test', verbose=False,
+                 num_mappers=None):
+        self.conn = self.get_connection(conn_id)
+        print(self.conn.extra)
```

Comment:
The `print` is indeed a debug line, I will remove it.

Also @hgrif thank you for pointing out the `test` id, this should be `default`. 👍